CMakeLists.txt

```diff
@@ -268,13 +356,18 @@ else()
 endif()
 
 if(USE_CUDA AND FIRST_CUDA)
-  include(cmake/ChooseBlas.cmake)
   include(3rdparty/mshadow/cmake/Utils.cmake)
   include(cmake/FirstClassLangCuda.cmake)
   include_directories(${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
 else()
   if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/3rdparty/mshadow/cmake)
+    # Workaroud to prevent mshadow from processing BLAS libraries
+    # BLAS libraries for MXNet are setup in cmake/ChooseBLAS.cmake
```

Comment:
Can we add a bit more info on what's the problem here with mshadow? shall we change mshadow as well?

Reply:
Done